scripts/extract_variables_from_integration_tests.py

```diff
@@ -0,0 +1,92 @@
+"""
+extracts manageiq integration test provider data/credentials
+into variables files for the wrapanapi testsuite
+
+this script uses manageiq integration test domain knowledge
+and should be executed using its virtualenv
+"""
+
+import click
+import six
+import pathlib2
+import json
+
+
+WANTED_TAGS = frozenset(['rhevm', 'ms_scvmm'])
+
+
+def extract_rhevm_system(mgmt, credentials):
```

Comment:
**OPTIONAL:** Consider combining the two system extract functions because they are very similar and the scvmm one seems to be a superset of rhevm.